.github/workflows/test.yml

```
@@ -11,7 +11,10 @@ on:
     paths-ignore:
       - "docs/**"
       - "**/README.md"
-  pull_request: {}
+  pull_request: 
+    paths-ignore:
+      - "docs/**"
+      - "**/README.md"
```

Should we go ahead an ignore all markdown files? 
```suggestion
      - "**/*.md"
```